### ISSUE TITLE
Case of --cache-control not working out

### ISSUE THREAD

**Issue by benjamin-chang** [apancutt/deploy-aws-s3-cloudfront#30]
Have the following in a package.json script...

```
deploy-aws-s3-cloudfront --source ./public --cache-control '**index.html:no-cache' '**.js:max-age=31536000,immutable' '**:max-age=604800' --non-interactive --soft-delete --invalidation-path '/*'
```

Which I'm calling with `npm run deploy -- --bucket <foo>`

The `cache-control` argument is not parsed. And the S3 objects do not have the associated metadata. 

node@v14.18.1
deploy-aws-s3-cloudfront@3.5.0

**Comment by benjamin-chang** [comment]
I hacked around this by changing the `cache-control` option to `cacheControl` and updating package.json to reflect this.

**Comment by apancutt** [comment]
Thanks for reporting this. Appears to be an issue in yargs whereby the `coerce()` function does not receive the option value when using dash notation. Fixed in 3.5.1.
